src/control/controlsortfiltermodel.cpp

```diff
@@ -17,7 +17,7 @@ bool ControlSortFilterModel::sortDescending() const {
 }
 
 void ControlSortFilterModel::sortByColumn(int column, bool descending) {
-    const bool oldColumn = sortColumn();
+    const int oldColumn = sortColumn();
```

Comment:
This is actually a bug that also affects 2.3. Please rebase te branch and retarget the PR.

Reply:
I think this code exist only in Main

Reply:
:see_no_evil: Correct. I checked out the wrong branch.